### ISSUE TITLE
Cannot run my circleci workflow with latest (or older) orb versions

### ISSUE THREAD

**Issue by andreacab** [CircleCI-Public/terraform-orb#36]
**Orb Version**
2.0.0
1.1.0

**Describe the bug**
Everything was working fine I believe 24 hours ago. Today is another day, and the job `validate` started failing at the `init` stage as it uses the `module_path` as an argument and not as the global option `-chdir`. Some people have noticed that too and two PRs could be fixing this issue I believe.
Because I can't control the TF version to be used with the executor used for the various orb jobs, I am just at the mercy of the terraform orb and terraform:light docker image combo. If the combination is a failing one, so are my workflows.

**To Reproduce**
1. Create a workflow
2. Add the orb job `validate` to that workflow
3. run it

**Expected behavior**
I would expect it to work or give me a chance to set the tag of the docker image used as the executor so that when using an older orb version I can tell it to use the according TF version.

**Comment by andreacab** [comment]
Closing as this has been temporarily fixed in 2.0.1

**Comment by vallard** [comment]
As this wasn't explicitly called out, we fixed this by locking down our terraform version which we hadn't before today when everything broke... 
```
jobs:
  tf: 
    executor: 
      name: terraform/default
      image: hashicorp/terraform:0.14.6
     ...
```
